docs/02_getting_started.md

```diff
@@ -37,14 +37,14 @@ You may even find some new, indispensable tools to help with continuous integrat
 
 **Git is:**
 
-- a distributed version control system or DVCS.
-- free and open source.
-- designed to handle everything from small to very large projects with speed and efficiency.
-- easy to learn and has a tiny footprint with lightning fast performance.
+- A distributed version control system (DVCS)
+- Free and open source
+- Designed to handle everything from small to very large projects with speed and efficiency
+- Easy to learn and has a tiny footprint with lightning fast performance
 
 Git features cheap local branching, convenient staging areas, and multiple workflows.
 
-As we begin to discuss Git (and what makes it special) it would be helpful if you could forget everything you know about other version control systems (VCSs) for just a moment. Git stores and thinks about information very differently than other VCSs.
+As we begin to discuss Git (and what makes it special) it would be helpful if you could forget everything you know about other version control systems (VCS's) for just a moment. Git stores and thinks about information very differently than other VCS's.
```

Comment:
I believe this should remain without apostrophes; no possession.

Reply:
Oops! I should have explained myself here. Before I made this change, I checked did some Google’ing and came across this: https://english.stackexchange.com/a/921

So that’s what drove the change, but after re-reviewing the [GitHub Brand Guide](https://brand.github.com/content/grammar), it agrees with your point. Thanks, @ericalloyd! 😊

Reply:
> omit the apostrophe, unless there are periods in the abbreviation

TIL! 🏆